internal/secrets/driver/driver.go

```diff
@@ -29,3 +29,7 @@ type Crypter interface {
 	// Encrypt encrypts the plaintext and returns the cipher message.
 	Encrypt(ctx context.Context, plaintext []byte) ([]byte, error)
 }
+
+// CrypterOptions controls Crypter behaviors.
```

Comment:
Don't think we need this; it's not used...? We want every public constructor to take `Options`, but it should probably be a different type per provider impl.

Reply:
@shantuo can you provide some context (a summary of in-person conversation?) for this being resolved?